components/brave_wallet_ui/components/desktop/popup-modals/add-account-modal/create-account-modal.tsx

```diff
@@ -224,79 +234,130 @@ export const CreateAccountModal = () => {
-                      data-value={network}
-                      key={network}
-                    >
-                      {BitcoinNetworkLocaleMapping[network]}
-                    </div>
-                  )
-                })}
-              </Select>
-            </SelectWrapper>
+            <Dropdown
+              value={bitcoinNetwork}
+              onChange={onChangeBitcoinNetwork}
+            >
+              <div slot='label'>
+                {getLocale('braveWalletAllowAddNetworkNetworkPanelTitle')}
+              </div>
+
+              <div slot='value'>
+                {BitcoinNetworkLocaleMapping[bitcoinNetwork]}
+              </div>
+
+              {BitcoinNetworkTypes.map((network) => {
+                return (
+                  <leo-option
+                    key={network}
+                    value={network}
+                  >
+                    {BitcoinNetworkLocaleMapping[network]}
+                  </leo-option>
+                )
+              })}
+            </Dropdown>
           )}
+
           {selectedAccountType?.coin === BraveWallet.CoinType.ZEC && (
-            <SelectWrapper>
-              <Select
-                value={zcashNetwork}
-                onChange={onChangeZCashNetwork}
-              >
-                {ZCashNetworkTypes.map((network) => {
-                  return (
-                    <div
-                      data-value={network}
-                      key={network}
-                    >
-                      {ZCashNetworkLocaleMapping[network]}
-                    </div>
-                  )
-                })}
-              </Select>
-            </SelectWrapper>
+            <Dropdown
+              value={zcashNetwork}
+              onChange={onChangeZCashNetwork}
+            >
+              <div slot='label'>
+                {getLocale('braveWalletAllowAddNetworkNetworkPanelTitle')}
+              </div>
+
+              <div slot='value'>{ZCashNetworkLocaleMapping[zcashNetwork]}</div>
+
+              {ZCashNetworkTypes.map((network) => {
+                return (
+                  <leo-option
+                    key={network}
+                    value={network}
+                  >
+                    {ZCashNetworkLocaleMapping[network]}
+                  </leo-option>
+                )
+              })}
+            </Dropdown>
           )}
+
           <Input
             value={accountName}
             placeholder={getLocale('braveWalletAddAccountPlaceholder')}
+            onInput={handleAccountNameChanged}
             onKeyDown={handleKeyDown}
-            onChange={handleAccountNameChanged}
-            autoFocus={true}
-          />
-
-          <NavButton
-            onSubmit={onClickCreateAccount}
-            disabled={isDisabled}
-            text={getLocale('braveWalletCreateAccountButton')}
-            buttonType='primary'
-          />
-        </StyledWrapper>
+            showErrors={isDisabled}
+            maxlength={30}
```

Comment:
BraveWallet.ACCOUNT_NAME_MAX_CHARACTER_LENGTH

Reply:
Thanks. I removed this prop value since the Nala component wasn't applying it
[a3e6c65](https://github.com/brave/brave-core/pull/22907/commits/a3e6c65516b717f2285997367093170880622356)